server/chat.ts

```diff
@@ -1847,15 +1847,23 @@ export const Chat = new class {
 
 	packageData: AnyObject = {};
 
+	getPluginName(file: string) {
+		const ext = pathModule.extname(file);
+		const folder = file.includes('chat-commands') ? 'chat-commands' : 'chat-plugins';
+		// the .slice(1, ...) removes the / from the beginning
+		let name = file.slice(file.indexOf(folder) + folder.length).slice(1, -ext.length);
```

Comment:
This will fail if PS is in a directory named `chat-commands`.

Reply:
Fixed, i believe.